pkg/codegen/prune.go

```diff
@@ -28,14 +28,14 @@ func walkSwagger(swagger *openapi3.T, doFn func(RefWrapper) (bool, error)) error
 
 	for _, p := range swagger.Paths {
 		for _, param := range p.Parameters {
-			walkParameterRef(param, doFn)
+			_ = walkParameterRef(param, doFn)
```

Comment:
This is bad, you should never do this to quash the lint error, just check the error, lots more below.

Reply:
Although yes, I 100% agree, the file so far has a lot of these, and I'd prefer us to come back and sort them

Reply:
I see a two options:
1. Disable `errcheck` linter in `.golangci-lint.yaml` for now.
2. Use https://github.com/deepmap/oapi-codegen/pull/845 which already fixes all of these.